mscore/scoreview.cpp

```diff
@@ -2595,11 +2594,14 @@ QVariant ScoreView::inputMethodQuery(Qt::InputMethodQuery query) const
                         else
                               r = toPhysical(text->canvasBoundingRect());
                         r.setHeight(r.height() + 10); // add a little margin under the cursor
-                        qDebug("   cursorRect: [%3f,%3f,%3f,%3f]", r.x(), r.y(), r.width(), r.height());
-                        return QVariant(r);
+                        qDebug("ScoreView::inputMethodQuery() updating cursorRect to: (%3f, %3f) + (%3f, %3f)", r.x(), r.y(), r.width(), r.height());
+                        return QVariant(r);     
                         }
                   case Qt::ImEnabled:
-                        return editMode();
+                        return true; // TextBase will always accept input method input
```

Comment:
without this change, the input method would only work the first time, but then stop working...I think cause when editMode() returned false upon unselecting a text element, it would get disabled.  So I found better to set true always when querying a textbox.